addons/l10n_it_edi/data/invoice_it_template.xml

```diff
@@ -84,7 +84,7 @@
                         <IscrizioneREA t-if="record.company_id.l10n_it_has_eco_index">
                             <!--1.2.4-->
                             <Ufficio t-esc="record.company_id.l10n_it_eco_index_office.code"/>
-                            <NumeroREA t-esc="record.company_id.l10n_it_eco_index_number"/>
+                            <NumeroREA t-esc="format_alphanumeric(record.company_id.l10n_it_eco_index_number[:20])"/>
```

Comment:
not related to the commit message

Reply:
I think it should be added in the commit message, because the xml checks if the string is not longer than...  so it can be useful.  Here, I don't know if it is such a good idea as it is a registration number and I don't suppose you want to give a shorter version of it.

Reply:
You're right I forgot to include that in the commit message :grimacing: 
@jco-odoo However it is necessary here, since if we are matching the spec as closely as possible, this is a clear case where the type is specified.
![image](https://user-images.githubusercontent.com/46533567/163968323-2b5704ee-5c43-4eeb-9260-c1c9cdd076d0.png)
and string20type is specified like this
![image](https://user-images.githubusercontent.com/46533567/163968346-3f9bc121-b3ee-4c55-a547-e2a08260d6d8.png)

Reply:
If it is an official registration number, is not it better that it would throw an error like it already does than shortening the number (risking providing the wrong one)

Reply:
It would be overkill and it has a size=20 (although not sure what it still does)

Reply:
I guess we can remove the slice since the field is size=20 anyway.